source/omitArgs.js

```diff
@@ -0,0 +1,50 @@
+import _curry2 from './internal/_curry2';
+
+/**
+ * Wraps a function of any arity (including nullary) in a function additional
+ * parameters that are omitted.
+ *
+ * @func
+ * @memberOf R
+ * @category Function
+ * @sig Number -> (* -> a) -> (* -> a)
+ * @param {Number} n The desired number of ignored parameters.
+ * @param {Function} fn The function to wrap.
+ * @return {Function} A new function wrapping `fn`.
+ * @see R.nAry
+ * @example
+ *
+ *      const takesTwoArgs = (a, b) => a + b;
+ *
+ *      takesTwoArgs(1, 2); //=> 3
+ *
+ *      const ignoresTwoArgs = R.omitArgs(2, takesTwoArgs);
```

Comment:
I would make `n` either 1 or 3 here, so that it's easily distinguised from the underlying functions's arity.